core/dbt/lib.py

```diff
@@ -92,26 +98,28 @@ def _get_operation_node(manifest, project_path, sql):
     )
 
     adapter = dbt.adapters.factory.get_adapter(config)
-    # TODO : This needs a real name?
-    sql_node = block_parser.parse_remote(sql, "name")
+    sql_node = block_parser.parse_remote(sql, node_name)
```

Comment:
@drewbanin Is this change from "name" to "query" (or custom) related to a change in core? I know we discussed this a bit but I haven't found record of it to confirm whether it's just for 1.3 or needs backporting as well

Reply:
Nope - we made this change to allow callers to provide a custom node name! The change from `name` to `query` is purely aesthetic -- `name` was a really terrible/confusing default :)

`_get_operation_node` is only called within this file, and all callers (eg. `compile_sql` provide this 3rd argument now)

Reply:
we should _definitley_ backport this part too. It's purely internal to the `lib.py` file, and it will give us the ability to use unique node names for concurrent compiles. Without this hook into configuring the model name, we're sort of limited in how we can concurrently make use of a manifest in memory

Reply:
Perfect, thank you!